jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -0,0 +1,255 @@
+
+Manually triggering a pipeline from the CircleCI web app is not available at this time.
+
+[#project-settings]
+== Project settings
+
+Within CircleCI, a project can have one or more **configurations**, which are pipeline definitions. _Configurations_ include, but are not limited to, a `.circleci/config.yml` file in your repo.
+
+A project can have one or more **triggers**, which are events from a source of change. _Triggers_ include, but are not limited to, a VCS. A trigger determines which configuration should be used to start a pipeline.
+
+The following settings are found by clicking the **Project Settings** button within your project. At this time, both configurations and triggers are limited to GitLab and GitHub App integrations.
+
+[#people]
+=== People
+
+Project roles give control over which users have access to which projects within an organization. This enables teams to have limited access to only their projects, while managers and others can have broader organizational access. The access options are:
+
+* Admin: Read and write access to the project and all settings and ability to manage other users' access.
+* Contributor: Read and write access to the project and some settings.
+* Viewer: Read only access to the project and some settings.
+
+For a complete list of permissions, see the xref:roles-and-permissions-overview#[Roles and permissions overview] page.
+
+[#configuration]
+=== Configuration
+
+You can add or delete a configuration source for your project. If you followed the steps above to connect GitHub, a GitHub configuration source has been automatically added for you.
+
+Once you define a configuration source, you can set up a trigger that points to that configuration.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-configuration.png[Configuration setup page]
+
+[#triggers]
+=== Triggers
+
+**The scheduled pipelines feature is not currently available for use with GitHub App integrations.** Triggers are described below, including how to use filters to trigger pipelines based on certain conditions.
+
+Add a trigger that specifies which configuration source starts a pipeline. If you followed the steps above to connect GitHub, a trigger set with GitHub as the configuration source has been automatically added for you.
+
+image::{{site.baseurl}}/assets/img/docs/gl-ga/gitlab-ga-project-settings-triggers.png[Trigger setup page]
+
+Triggers and trigger rules determine how CircleCI handles events from the source of change, in this case, GitHub.
+
+When a trigger is created, CircleCI registers a webhook with GitLab. Push events from GitLab are sent to CircleCI. CircleCI then uses the event data to determine _if_ a pipeline should run, and if so, _which_ pipeline should be run.
```

Comment:
With GitHub apps, there is no webhook created. When users install the app, GitHub starts sending events for the repositories they granted access to. When a trigger is created, CircleCI will then have enough information to use the event data to determine if a pipeline should run.